partner_contact_role/views/res_partner_role.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Camptocamp
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="res_partner_role_tree_view">
+        <field name="name">res.partner.role.tree</field>
+        <field name="model">res.partner.role</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom">
+                <field name="name" />
+                <field name="active" widget="boolean_toggle" />
+           </tree>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="res_partner_role_act_window">
+        <field name="name">Partner Role</field>
+        <field name="res_model">res.partner.role</field>
+        <field name="view_mode">tree</field>
```

Comment:
```suggestion
        <field name="view_mode">tree,form</field>
```
Someone might be depending on this module to extend the form view, so we will keep the form view